layouts/partials/header.html

```diff
@@ -3,7 +3,7 @@
 {{ $version := index $productPathData 1 }}
 
 <!doctype html>
-<html lang="en">
+<html lang="en" data-theme="light">
```

Comment:
Was this a debugging addition or is it meant to be permanent? While testing the theme switcher, this doesn't appear to change at all.